### PR TITLE
Add lightweight blog guidelines

### PR DESCRIPTION
**Pull request by celestehorgan**
Signed-off-by: Celeste Horgan <celeste@cncf.io>

### CODE REVIEW

File: ws-marketing/blog-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Inclusive Naming Initiative Blog Guidelines #
+
+## Overview ##
+
+The Inclusive Naming Initiative blog serves as a channel INI member projects to share content with the community at large, as well as a home for press releases.
```
Comment (xmulligan) [inline ws-marketing/blog-guidelines.md:5]:
```suggestion
The Inclusive Naming Initiative's blog serves as a channel for INI to share content with the community at large, as well as a home for press releases.
```

File: ws-marketing/blog-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The content on our blog consists of:
+* Articles about projects related to the Inlcusive Naming Initiative's work 
+* Use cases and success stories from
```
Comment (xmulligan) [inline ws-marketing/blog-guidelines.md:9]:
```suggestion
* Use cases and success stories from people, projects, and organizations using INI's work
```

File: ws-marketing/blog-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Inclusive Naming Initiative Blog Guidelines #
+
+## Overview ##
+
+The Inclusive Naming Initiative blog serves as a channel INI member projects to share content with the community at large, as well as a home for press releases.
+
+The content on our blog consists of:
+* Articles about projects related to the Inlcusive Naming Initiative's work
```
Comment (xmulligan) [inline ws-marketing/blog-guidelines.md:8]:
```suggestion
* Articles about projects related to the INI's work 
```

File: ws-marketing/blog-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The Inclusive Naming Initiative blog serves as a channel INI member projects to share content with the community at large, as well as a home for press releases.
+
+The content on our blog consists of:
+* Articles about projects related to the Inlcusive Naming Initiative's work 
+* Use cases and success stories from 
+
+Other topics are welcome, but it needs to have a direct link to the Inclusive Naming Initiative's work.
+
+We accept content from INI members and their organizations. The content needs to be originally published within the last 2 months.
```
Comment (xmulligan) [inline ws-marketing/blog-guidelines.md:13]:
```suggestion
We accept content from INI members and their organizations. The content can be original or published within the last 2 months.
```

File: ws-marketing/blog-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+
+## Content ##
+
+The Inclusive Naming Initiative blog audience is developers, open source maintainers, and other people managing codebases. Keep this in mind as you develop your blog content.
```
Comment (xmulligan) [inline ws-marketing/blog-guidelines.md:18]: I would argue that it is much larger than this

File: ws-marketing/blog-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Some things to think about:
+
+* While we encourage our member projects to submit use cases and other stories, the Inclusive Naming Initiative blog is not intended as an advertising channel.
```
Comment (xmulligan) [inline ws-marketing/blog-guidelines.md:22]:
```suggestion
* While we encourage our members to submit use cases and other stories, the Inclusive Naming Initiative blog is not intended as an advertising channel.
```

File: ws-marketing/blog-guidelines.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+* The most interesting posts are implementation case studies or other practical content to help developers make naming changes.
+
+Members can add 1-2 sentences at the end of their blog with a link to an external site such as their website or a gated asset. Examples for best practices:
```
Comment (xmulligan) [inline ws-marketing/blog-guidelines.md:26]:
```suggestion
Members can add 1-2 sentences at the end of their blog with a link to an external site such as their website. Examples for best practices:
```

File: ws-marketing/blog-guidelines.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+* Links to projects websites or Github
+
+INI will review the statement, as we do for the content for the blog, to approve the content. If changes are needed, we will provide suggested changes. 
+
+## Promotion ##
+Your blog will be shared on INI's Twitter account. Please feel free to retweet or share.
+
+## How to submit for consideration ##
+
+Please submit a brief summary and the topic of the post to pr@inclusivenaming.org for consideration. Or you can supply the blog for review. We will respond with a proposed date of publication or if it’s not suitable for the INI blog, we will provide feedback and direction.
```
Comment (xmulligan) [inline ws-marketing/blog-guidelines.md:38]:
```suggestion
Please submit a brief summary and the topic of the post to pr@inclusivenaming.org for consideration. Or you can supply the blog for review by creating a PR in the [blog folder](https://github.com/inclusivenaming/website/tree/main/content/blog/_posts). We will respond with a proposed date of publication or if it’s not suitable for the INI blog, we will provide feedback and direction.
```

File: ws-marketing/blog-guidelines.md (hunk 3 above)
Comment (edwarnicke) [reply ws-marketing/blog-guidelines.md:8]: Perhaps:
```suggestion
* Articles about projects related to the INI's mission 
```

File: ws-marketing/blog-guidelines.md (hunk 2 above)
Comment (edwarnicke) [reply ws-marketing/blog-guidelines.md:9]: Perhaps:

```suggestion
* Use cases and success stories from people, projects, and organizations furthering INI's mission
```

File: ws-marketing/blog-guidelines.md (hunk 4 above)
Comment (edwarnicke) [reply ws-marketing/blog-guidelines.md:13]: Perhaps:

```suggestion
We accept content in support of INI's mission. The content can be original or published within the last 2 months.
```

File: ws-marketing/blog-guidelines.md (hunk 5 above)
Comment (edwarnicke) [reply ws-marketing/blog-guidelines.md:18]: @xmulligan Agree... but not sure how to word it :)

File: ws-marketing/blog-guidelines.md (hunk 6 above)
Comment (edwarnicke) [reply ws-marketing/blog-guidelines.md:22]: Perhaps:

```suggestion
* While we encourage the submission of use cases and other stories, the Inclusive Naming Initiative blog is not intended as an advertising channel.
```

File: ws-marketing/blog-guidelines.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+* While we encourage our member projects to submit use cases and other stories, the Inclusive Naming Initiative blog is not intended as an advertising channel.
+
+* The most interesting posts are implementation case studies or other practical content to help developers make naming changes.
+
+Members can add 1-2 sentences at the end of their blog with a link to an external site such as their website or a gated asset. Examples for best practices:
+* Share contact info of  speaker - twitter, slack, email
+* Links to company websites
+* Links to projects websites or Github
+
```
Comment (edwarnicke) [inline ws-marketing/blog-guidelines.md:30]: Question... do we want to encourage or provide for the inclusion of an author photo?

Do we want provide for the inclusion of a 'blog image' that can be visually associated with the blog post (helps with things like tweets etc)?

**Pull request merged**